### PR TITLE
Updates to install/uninstall scripts.

### PR DESCRIPTION
**Pull request by nostrus-dominion**
I've been using this project on and off for a little over a  year now and I wanted to contribute to the project in a small way.

I made some changes to these scripts to make them more readable and give it a bit of flair. The big thing is change a lot of how the confirmations work and what the user has to do to make sure they are doing what they mean to be doing.

For example to continue the script after making a selection for the installation is to type out "yes" during the prompt. This way the user isn't just mashing ENTER to get things going.

I also tried to make things very clear on what needs to be done after the installation in terms of R/W permissions for the directories that is needed for these services to work.  Maybe we can work on adding ACL or chown later down the road?

Anyway I hope I can help out in the future and if you have any questions let me know.

### CODE REVIEW

File: servarr/servarr-install-script.sh [hunk 1]
```diff
@@ -29,22 +30,44 @@
...
 fi
 
-echo "Select the application to install: "
+### Title Splash
+echo -e ${brown}
+echo -e "#############################################################"
+echo -e "#                                                           #"
+echo -e "#   Welcome to the Servarr Community Installation Script!   #"
+echo -e "#                                                           #"
+echo -e "#  This script is for the Debian / Ubuntu / Raspbian users  #"
```
Comment (bakerboy448) [inline servarr/servarr-install-script.sh:59]:
```suggestion
echo -e "#  This script is for the Debian-based distro users  #"
```

File: servarr/servarr-uninstall-script.sh [hunk 2]
```diff
@@ -9,45 +11,125 @@
...
-# Default application name; override by enter manually
-select app in "lidarr" "prowlarr" "radarr" "readarr" "whisparr" "manual" "quit "; do
-
+### Title Splash
+echo -e ${brown}
+echo -e "   #############################################################"
+echo -e "   #                                                           #"
+echo -e "   #   Welcome to the Servarr Community UNInstaller Script!    #"
+echo -e "   #                                                           #"
+echo -e "   #  This script is for the Debian / Ubuntu / Raspbian users  #"
```
Comment (bakerboy448) [inline servarr/servarr-uninstall-script.sh:40]: same as install

File: servarr/servarr-uninstall-script.sh [hunk 3]
```diff
@@ -56,42 +138,108 @@ Group=$(systemctl show -p Group --value "$app.service")
...
+    echo -e ${yellow}"Data directory deleted."${reset}
+    sleep 2
+fi
+
+# User confirmation remove $User
+echo ""
+read -p "Do you want to remove user '$User'? (y/N): " deluser
+deluser=${deluser:-N}
+
+if [[ $deluser =~ ^[Yy]$ ]]; then
```
Comment (bakerboy448) [inline servarr/servarr-uninstall-script.sh:199]: let's require "delete USER" as the acknowledgement here

File: servarr/servarr-uninstall-script.sh [hunk 4]
```diff
@@ -56,42 +138,108 @@ Group=$(systemctl show -p Group --value "$app.service")
...
+    echo -e "There are still users in the group '$Group':" ${yellow}$users_in_group${reset}
+    sleep 1
+    echo ""
+    echo -e "Group '$Group' ${red}CANNOT${reset} be removed until all users are removed from it."
+    echo ""
+    echo "If this user is no longer in use, then as root use 'userdel $users_in_group' to delete the user from your system."
+    sleep 1
+else
+    echo ""
+    read -p "Do you want to remove group '$Group'? (y/N): " delgroup
```
Comment (bakerboy448) [inline servarr/servarr-uninstall-script.sh:223]: same require "delete group"

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
Thanks this looks really good; couple tweaks!

**Comment by nostrus-dominion** [comment]
I'll get on the other things shortly but I'm in the middle of another project. Should be ready by 1700 ET

**Review by bakerboy448 — APPROVED** [review]

**Comment by nostrus-dominion** [comment]
Is there anything else that would be good to work on?

**Comment by bakerboy448** [comment]
applying similar changes upstream to Sonarr's install script and maybe adding an uninstall script in Sonarr/Sonarr repo would be about all that comes to mind

**Pull request merged**